src/Process/Installer.php

```diff
@@ -150,20 +150,15 @@ public function run()
      */
     public function getProcess()
     {
-        switch ($this->type) {
-            case 'github':
-            case 'github-https':
-            case 'bitbucket':
-                if ($this->tree) {
-                    $process = $this->installViaSubtree();
-                }
-
-                $process = $this->installViaGit();
-                break;
+        if ($this->type) {
+            if ($this->tree) {
+                $process = $this->installViaSubtree();
+            }
 
-            default:
-                $process = $this->installViaComposer();
-                break;
+            $process = $this->installViaGit();
+        }
+        else {
+            $process = $this->installViaComposer();
```

Comment:
Else no more needed if the previous early return, this can be returned directly too.